src/engine/GameObjects/MapNode.cxx

```diff
@@ -68,6 +68,32 @@ void MapNode::setTileID(const std::string &tileID)
   }
 }
 
+bool MapNode::checkBuildingLayerSlopePlaceable(const std::string &tileID, const Layer &layer) const
```

Comment:
This function seems to be checking if a buildings tiledata has slopeTiles and if it does it's allowing it to be built on slopes, if it doesn't then it cant be built on slopes, right? I tested this out and it works great but could we get the same thing happening for cornerTiles also?

Reply:
Check it, it also works on corner tiles

Reply:
Have you pushed your changes?